docs/source/using/profiles.adoc

```diff
@@ -43,16 +61,17 @@ $ minishift profile list
 [[switching-profiles]]
 == Switching Profiles
 
-With the help of xref:../command-ref/minishift_profile_set.adoc#[`minishift profile set`] you can switch between profiles or set a profile as active.
-When a profile is active, it is not required to pass `--profile` flag to run minishift commands against that profile.
+With the help of xref:../command-ref/minishift_profile_set.adoc#[`minishift profile set`] you can switch between profiles.
+However only one profile can be active at any time.
+When the profile is active, you do not need to use `--profile` flag to run `minishift` commands against the active profile.
```

Comment:
"When the profile is active" is redundant -- the sentence reads well without it: "You do not need to use the `--profile` flag to run `minishift` commands against the active profile."